src/remote-checkout/remote-checkout-request-sender.ts

```diff
@@ -57,6 +57,12 @@ export default class RemoteCheckoutRequestSender {
 
         return this._requestSender.post(url, { timeout, headers: SDK_VERSION_HEADERS });
     }
+
+    cancelToken(provider: string, token: string, { timeout }: RequestOptions = {}): Promise<Response<any>> {
+        const url = `/remote-checkout/${provider}/cancel-client-token/${token}`;
```

Comment:
@davidchin this is what we were discussing, does this seem correct to add another method to remote checkout. Seems like we might need to make this implicit API public 🤔

Reply:
Yes, likely not a good idea to add another private / undocumented endpoint if possible. Also, I believe APIs accessible via the storefront should be under the storefront namespace, as they have HTTPS/CSRF/CORS checks out-of-the-box. @BC-PabloRivera @rabinshr, is there a particular reason why `/remote-checkout` namespace is chosen?